cmake/modules/FindCGAL.cmake

```diff
@@ -36,7 +36,7 @@ IF(DEAL_II_HAVE_CXX17)
 ELSE(DEAL_II_HAVE_CXX17)
   SET(CGAL_FOUND FALSE)
   SET(CGAL_INCLUDE_DIRS "-NOTFOUND")
-  MESSAGE("-- WARNING: CGAL wrappers require cxx17. Disabling CGAL Support.")
+  MESSAGE(WARNING "CGAL wrappers require C++17. Disabling CGAL Support.")
```

Comment:
I really don't mind if you want to keep this as it was, or change to `MESSAGE(STATUS`. I just saw the formatting, and recognised that the `--` and indentation could be done automatically by CMake.